index.js

```diff
@@ -4,6 +4,7 @@ const path = require('path');
 const resolve = require('resolve');
 const Funnel = require('broccoli-funnel');
 const MergeTrees = require('broccoli-merge-trees');
+const BroccoliDebug = require('broccoli-debug');
```

Comment:
Is this something you added for testing this out, to be removed later? Or to keep in to help consumers?

Reply:
I was thinking to keep it, there's no difference for consuming apps, other addons do this too.

It just helps debugging easily the build tree. For example running `BROCCOLI_DEBUG=ember-polaris:* ember b` will log the trees at each step and put it in a `DEBUG` folder.